operations/security_and_authentication/Add_LDAP_User_Federation.md

```diff
@@ -205,59 +246,173 @@ LDAP user federation is not currently configured in Keycloak. For example, if it
+
+      ```bash
+      ncn-m001# /mnt/pitdata/${CSM_RELEASE}/hack/load-container-image.sh dtr.dev.cray.com/zeromq/zeromq:v4.0.5
+      ```
+
+   3. Re-encrypt the existing secrets:
+
+      ```bash
+      ncn-m001# /mnt/pitdata/prep/site-init/utils/secrets-reencrypt.sh customizations.yaml \
+      /mnt/pitdata/prep/site-init/certs/sealed_secrets.key /mnt/pitdata/prep/site-init/certs/sealed_secrets.crt
+      ```
+      
+4. Encrypt the static values in the customizations.yaml file after making changes.
+
+   The following command must be run within the site-init directory.
+
+   ```bash
+   ncn-m001# ./utils/secrets-seed-customizations.sh customizations.yaml
+   ```
+
+   Expected output looks similar to:
+
+      ```bash
+      Creating Sealed Secret keycloak-certs
+      Generating type static_b64...
+      Creating Sealed Secret keycloak-master-admin-auth
+      Generating type static...
+      Generating type static...
+      Generating type randstr...
+      Generating type static...
+      Creating Sealed Secret cray_reds_credentials
+      Generating type static...
+      Generating type static...
+      Creating Sealed Secret cray_meds_credentials
+      Generating type static...
+      Creating Sealed Secret cray_hms_rts_credentials
+      Generating type static...
+      Generating type static...
+      Creating Sealed Secret vcs-user-credentials
+      Generating type randstr...
+      Generating type static...
+      Creating Sealed Secret generated-platform-ca-1
+      Generating type platform_ca...
+      Creating Sealed Secret pals-config
+      Generating type zmq_curve...
+      Generating type zmq_curve...
+      Creating Sealed Secret munge-secret
+      Generating type randstr...
+      Creating Sealed Secret slurmdb-secret
+      Generating type static...
+      Generating type static...
+      Generating type randstr...
+      Generating type randstr...
+      Creating Sealed Secret keycloak-users-localize
+      Generating type static...
+      ```
+
+5. Re-apply the cray-keycloak-users-localize Helm chart with the updated customizations.yaml file.
+
+    1.  Determine the cray-keycloak-users-localize chart version that is currently deployed.
+
+        ```bash
+        ncn-m001# helm ls -A -a | grep cray-keycloak-users-localize
         ```
 
-    2.  Encrypt the static values in the customizations.yaml file after making changes.
+    2.  Create a manifest file that will be used to reapply the same chart version.
 
         ```bash
-        ncn-w001# utils/secrets-seed-customizations.sh customizations.yaml
+        ncn-m001# cat << EOF > ./cray-keycloak-users-localize-manifest.yaml
+        apiVersion: manifests/v1beta1
+        metadata:
+          name: reapply-cray-keycloak-users-localize
+        spec:
+          charts:
+            - name: cray-keycloak-users-localize
+            namespace: services
+            values:
+              imagesHost: dtr.dev.cray.com
```

Comment:
Doesn't seem like it should be necessary to set this value since the cray-keycloak-users-localize chart doesn't use it.